src/Teams/Models/MultipleQuestionEditModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Teams.Models
+{
+    public class MultipleQuestionEditModel
+    {
+        public string QuestionText { get; set; }
+        public string[] TextArray { get; set; }
+        public bool[] CheckboxValueArray { get; set; }
+        public Guid id { get; set; }
```

Comment:
Please change Guid to int. Database cannot create clustered index with guid. And even u think clustered index is not a problem this is just a very strange to see guids instead of int

Reply:
There are many methods not written by me. Then I don't understand how they work as well. They are also used elsewhere in the project, so I think this may be a problem for refactoring. Correct me if I am wrong.